### PR TITLE
polkadot-v1.13.0

### PR DESCRIPTION
**Pull request by clangenb**
Noteworthy changes:
* The update enforces the runtimes and the collators to be async backing ready (even if it is deactivated). This means after a runtime upgrade, we can only produce blocks if the collator binary is updated.

Todo:
- [x] Double Check asset conversion changes

### CODE REVIEW

File: polkadot-parachains/integritee-runtime/Cargo.toml [hunk 1]
```diff
@@ -22,10 +22,11 @@ scale-info = { workspace = true }
 assets-common = { workspace = true }
 cumulus-pallet-aura-ext = { workspace = true }
 cumulus-pallet-dmp-queue = { workspace = true }
-cumulus-pallet-parachain-system = { workspace = true, features = ["parameterized-consensus-hook"] }
+cumulus-pallet-parachain-system = { workspace = true }
```
Comment (clangenb) [inline polkadot-parachains/integritee-runtime/Cargo.toml:25]: It aways enabled now as the runtimes need to be async backing ready

File: polkadot-parachains/integritee-runtime/src/lib.rs [hunk 2]
```diff
@@ -878,9 +879,21 @@ pub type NativeAndAssets = UnionOf<
...
+
+pub type AscendingLocator =
+	Ascending<AccountId, NativeOrWithId<AssetIdForTrustBackedAssets>, PoolIdToAccountId>;
+
+pub type WithFirstAssetLocator = WithFirstAsset<
+	Native,
+	AccountId,
+	NativeOrWithId<AssetIdForTrustBackedAssets>,
+	PoolIdToAccountId,
+>;
```
Comment (clangenb) [inline polkadot-parachains/integritee-runtime/src/lib.rs:896]: Todo: I just added these to make the code compile: we have to check if this is a sane config.

File: polkadot-parachains/integritee-runtime/src/weights/pallet_xcm.rs [hunk 3]
```diff
@@ -394,6 +394,4 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
             .saturating_add(T::DbWeight::get().reads(1))
             .saturating_add(T::DbWeight::get().writes(1))
     }
-    fn execute_blob() -> Weight { todo!() }
-    fn send_blob() -> Weight { todo!() }
```
Comment (clangenb) [inline polkadot-parachains/integritee-runtime/src/weights/pallet_xcm.rs:398]: Dangerous XD

Comment (brenzi) [reply polkadot-parachains/integritee-runtime/src/weights/pallet_xcm.rs:398]: yes: #267

File: polkadot-parachains/integritee-runtime/src/lib.rs (hunk 2 above)
Comment (brenzi) [reply polkadot-parachains/integritee-runtime/src/lib.rs:896]: will test the DEX flow like here: #279 & #285

**Comment by brenzi** [comment]
testing AssetConversion setup:
* register ROC
* XCM ROC to integritee
* create pool (ROC, TEER)
* swap TEER <> ROC
all worked. therefore, the AssetConversion config appears sound

**Review by brenzi — APPROVED** [review]

**Pull request merged**